### ISSUE TITLE
enhacements for theme-update.sh

### ISSUE THREAD

**Issue by gnadelwartz** [qooob/authentic-theme#747]
with theme update you get always the lastest repository changes, thats what it should do.
nevertheless I have two suggestions to improve it:

- update in non default dir name
  I wanted to update authentic-theme even its in a different folder, i.e. authentic-theme-test
  this failed because you assume the folder name IS authentic-theme. I solved this by changing the following line
`DIR="$(echo $DIR | sed 's/\/authentic-theme*//g')"` (note the star)

- update to latest release
  your script always updates to last development state, but I wanted to update to latest release
  because the development version may be broken from time to time. my solution is to copy
  theme-update.sh to theme-release.sh and change the following lines:
```
latest=`curl  -s -L https://github.com/qooob/authentic-theme/releases/latest | sed -n '/\<h4>Version / s/.\*\<h4>Version \([^ ]*\).*/\1/p`

echo -e "\e[49;1;34;182mPulling in latest RELEASE ${latest} for\e[0m \e[49;1;37;182mAuthentic Theme\e[0m (https://github.com/qooob/authentic-theme)..."
git clone --depth 1 --branch ${latest} https://github.com/qooob/authentic-theme.git "$DIR/.~authentic-theme"
```

If you are interested I can provide a pull request ...

**Comment by qooob** [comment]
Hi, Kay.

I will take a look, thanks. No pull request needed, I'll do it.

No other script needed, parameter is enough.

**Comment by qooob** [comment]
`DIR="$(echo $DIR | sed 's/\/authentic-theme*//g')"` is incorrect. It should be `.*`.

What is the point in different directory?

**Comment by gnadelwartz** [comment]
oops, you are right :-)

like my Problem with user min, see #748 , I switch between original theme version delivered with usermin and development version by simply renaming the dir.

from time to time I want to test the newest version an run theme-update.sh to get the newest changes
if I do this in the renamed dir script is not working. with the proposed change I can run theme-update.sh as long as dir name starts with authentic-theme.

**Comment by qooob** [comment]
Oh, if you mean that you want to choose the theme when it's in other directory - it will not work. Some paths are hardcoded. 

My recomendation - don't do any tests before you try updated version. This is what I do quite a bit.

In the future, running `theme-update.sh -release` will let you roll back to always working release.

**Comment by gnadelwartz** [comment]
nice Thanks!

**Comment by gnadelwartz** [comment]
don't know fi you get notifications about closed issues, found my problem with #748 but have no clue how to resolve ...

**Comment by qooob** [comment]
https://github.com/qooob/authentic-theme/commit/3f941ce44f1717f01017faba5b166043b35950a9
